index.html

```diff
@@ -0,0 +1,390 @@
+      <section class="text-box2">
+        <div class="text-info2">
+          <h2 class="text1a">
+            I can help you build a product , feature or website Look through
+            some of my work and
+          </h2>
+          <h3 class="text-info2b">
+            experience! If you like what you see and have a project you need
+            coded,
+          </h3>
+          <h4 class="text-info3c">don’t hestiate to contact me.</h4>
+        </div>
+      </section>
+    </div>
+
+    <button type="submit" class="resume-btn">Get My Resume</button>
+
+    <section class="new-end-section">
+      <ul class="about-me-end-section">
+        <li>
+          <div class="type-of-cards">
+            <div class="cat-images">
+              <img src="ball.svg" alt="picture of globe" />
+            </div>
+
+            <h2 class="cat-title">Languages</h2>
+
+            <ul class="cat-things">
+              <li>JavaScript</li>
+              <li>Ruby</li>
+              <li>Html</li>
+              <li>CSS</li>
+            </ul>
+          </div>
+        </li>
+
+        <li>
+          <div class="type-of-cards">
+            <div class="cat-images">
+              <img src="frame.svg" alt="picture of globe" />
+            </div>
+
+            <h2 class="cat-title">Frameworks</h2>
+            <div>
+              <ul class="cat-things">
+                <li>Bootstrap</li>
+                <li>Ruby on rails</li>
+                <li>RSpec</li>
+                <li>Capybara</li>
+                <li>Selenium</li>
+              </ul>
+            </div>
+          </div>
+        </li>
+
+        <li>
+          <div class="type-of-cards">
+            <div class="cat-images">
+              <img src="ball.svg" alt="picture of globe" />
+            </div>
+
+            <h2 class="cat-title">Skills</h2>
+
+            <ul class="cat-things">
+              <li>Codekit</li>
+              <li>Github</li>
+              <li>CodePen</li>
+              <li>GitLab</li>
+              <li>Terminal</li>
+            </ul>
+          </div>
+        </li>
+      </ul>
+    </section>
+
+    <footer class="contactpage">
+      <section class="projects-end">
+        <h2 class="textline1">I'm always</h2>
+        <h3 class="textline2">interested in</h3>
+        <h4 class="textline3">hearing about</h4>
+        <h5 class="textline4">new projects, so</h5>
+        <h6 class="textline5">if you'd like to</h6>
+        <h6 class="textline6">chat please get</h6>
+        <h6 class="textline7">in touch</h6>
+      </section>
+      <section class="projects-2-big">
+        <div class="new-paragraph">
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </div>
+      </section>
+
+      <section class="form-section">
+        <div class="form-container">
+          <form action="https://formspree.io/f/myyvzkag" method="post">
+            <ul class="form-info">
```

Comment:
[OPTIONAL] It would be nice if you could brush up the layout of your form to make everything centered correctly and create proper spacing between elements.

![image](https://user-images.githubusercontent.com/38088818/188007817-708a30ea-3825-4817-b3ad-0a683867a72a.png)

Reply:
@KDlamini thanks i just fixed it